topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -371,47 +522,81 @@ The quality of the previous dataset was pretty good from the beginning and we im
 > ### {% icon comment %} Bad quality sequences
 > If the quality of the reads is not good, we should always first check what is wrong and think about it: it may come from the type of sequencing or what we sequenced (high quantity of overrepresented sequences in transcriptomics data, biaised percentage of bases in HiC data).
```

Comment:
```suggestion
> If the quality of the reads is not good, we should always first check what is wrong and think about it: it may come from the type of sequencing or what we sequenced (high quantity of overrepresented sequences in transcriptomics data, biased percentage of bases in HiC data).
```